### DISCUSSION TITLE
export / transcompile to c?

### DISCUSSION THREAD

**Discussion by goyalyashpal** [c3lang/c3c#1133] [Q&A] [answered]
hi again!

is there some facility available to [transcompile] c3 code into c code? i read somewhere in the documentation that c3 compiler does that internally but searching the docs for that, i couldn't find it now.

[transcompile]: https://en.wiktionary.org/wiki/transcompile "transpile = Shortening of transcompile"

**Comment by data-man** [comment]
> c3 compiler does that internally

C3 is not Nim or V. :)

**Reply by goyalyashpal** [reply]
ahw, it would have been nice to do that. as there are some places where c is used. being able to write in c3, and submit in c would have been awesome (:

but anyhow, thanks for prompt response 😊

**Comment by lerno** [comment, marked as answer]
No, but you can compile a static library in C3 and export C headers for it.

**Reply by ccll** [reply]
IMHO this is not the point, the best thing of transpiling to C is automatic platform support, almost every platform in the world have a C89/C99 toolchain, I really would like to use c3 programming for iOS/android, TVs, game consoles, network routers...
